src/ConversationList/ConversationListItem.vala

```diff
@@ -53,6 +53,7 @@ public class Mail.ConversationListItem : VirtualizingListBoxRow {
         weak Gtk.StyleContext messages_style = messages.get_style_context ();
         messages_style.add_class (Granite.STYLE_CLASS_BADGE);
         messages_style.add_class (Gtk.STYLE_CLASS_SIDEBAR);
```

Comment:
I guess we can remove this, since it's probably a workaround for what flat class is intended to do